src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -230,7 +230,7 @@ protected StorageEnvironmentOptions(VoronPathSetting tempPath, IoChangesNotifica
 
             PrefetchSegmentSize = 4 * Constants.Size.Megabyte;
             PrefetchResetThreshold = 8 * (long)Constants.Size.Gigabyte;
-            JournalsSizeThreshold = 1 * (long)Constants.Size.Gigabyte;
+            JournalsFilesAmountThreshold = 1 * (long)Constants.Size.Gigabyte;
```

Comment:
`1 * (long)Constants.Size.Gigabyte;` sure ? we switched from bytes threshold to number of files